.github/workflows/regression-test-auto.yml

```diff
@@ -0,0 +1,76 @@
+name: Regression Tests - Agglayer
+'on':
+  pull_request:
+    types: [opened, synchronize]  # Trigger on new pull requests and when existing ones are synchronized
+    branches:
+      - main
+    inputs:
+      bake_time:
+        description: bake time (minutes)
+        required: false
+        default: 20
+jobs:
+  deploy_devnet:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Docker
+        uses: docker/setup-buildx-action@v1
+      - name: Clone and build agglayer image locally
+        run: |
+          git clone https://github.com/0xPolygon/agglayer.git
+          cd agglayer
+          if [ "$GITHUB_EVENT_NAME" == "pull_request" ]; then
+              GITHUB_SHA=$(cat $GITHUB_EVENT_PATH | jq -r .pull_request.head.sha)
+          fi
+          git checkout $GITHUB_SHA
+          echo "Containerizing and testing commit: $GITHUB_SHA"
+          docker compose -f docker/docker-compose.yaml build --no-cache agglayer
+      - name: Install Foundry
+        uses: foundry-rs/foundry-toolchain@v1
+      - name: Clone internal kurtosis-cdk repo
+        run: |
+          git clone https://github.com/0xPolygon/kurtosis-cdk.git
+          cd kurtosis-cdk
+          git checkout dan/jit_containers
```

Comment:
Are we going to merge this?

Reply:
Yup it's not needed anymore. We would just need to install `yq` and replace the `zkevm_agglayer_image` value by the name of the docker image (e.g. `agglayer:local`).

For reference: https://github.com/0xPolygon/kurtosis-cdk/blob/main/.github/workflows/regression-tests.yml#L102-L115

Reply:
fixed!